components/gitpod-db/src/typeorm/auth-code-repository-db.ts

```diff
@@ -56,9 +56,13 @@ export class AuthCodeRepositoryDB implements OAuthAuthCodeRepository {
         };
     }
     public async persist(authCode: DBOAuthAuthCodeEntry): Promise<void> {
```

Comment:
for the reference, `persist` is called by the `oauth2-server` library and may have no error handling for this case.